
uint8 ADD=0
uint8 MODIFY=0
uint8 DELETE=1

int32 action                 # 0 add/modify objects, 1 deletes objects
ObjectState[] object_states  # the state data
